README.md

```diff
@@ -1,46 +1,63 @@
-# Getting Started with Create React App and Redux
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app), using the [Redux](https://redux.js.org/) and [Redux Toolkit](https://redux-toolkit.js.org/) template.
+# React-Redux Capstone for Microverse
 
-## Available Scripts
+Web application that utilizes Riot's League of Legends API to display a list of character and alloq filtering through them, as well as seeing additional details for each character.
 
-In the project directory, you can run:
 
-### `npm start`
+## Built With
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+- HTML/CSS/JS
+- React
+- Redux
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+## Quick View
 
```

Comment:
- [ ] OPTIONAL
Kindly add instructions in your ReadMe on how to run the test cases of your project.